src/pages/project/project_wizard/EmptyProjectForm.tsx

```diff
@@ -35,10 +43,31 @@ export const EmptyProjectForm = ({ newProject, setNewProject, setShowProjectForm
           data-testid={dataTestId.newProjectPage.titleInput}
           variant="filled"
           onChange={(event) => setTitle(event.target.value)}
+          InputProps={{
+            endAdornment: titleSearch.isPending ? (
+              <CircularProgress size={20} />
+            ) : titleSearch.duplicateProject ? (
+              <ErrorIcon color="warning" />
+            ) : debouncedTitle && titleSearch.duplicateProject === undefined ? (
```

Comment:
Helt greit for meg om det blir sånn, bare vær obs på at om `duplicateProject` blir `null` for en eller annen grunn vil ikke dette virke igjen
```suggestion
            ) : debouncedTitle && !titleSearch.duplicateProject ? (
```